packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/pages/ListView/tests/ListView.test.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { render, waitFor } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+import { Provider } from 'react-redux';
+import { QueryClientProvider, QueryClient } from 'react-query';
+import userEvent from '@testing-library/user-event';
+import { rest } from 'msw';
+import { setupServer } from 'msw/node';
+import { ThemeProvider, lightTheme } from '@strapi/design-system';
+import { MemoryRouter } from 'react-router-dom';
+
+import configureStore from '../../../../../../../../../admin/src/core/store/configureStore';
+import ReviewWorkflowsListView from '..';
+import { reducer } from '../../../reducer';
+
+jest.mock('@strapi/helper-plugin', () => ({
+  ...jest.requireActual('@strapi/helper-plugin'),
+  // eslint-disable-next-line react/prop-types
+  CheckPagePermissions({ children }) {
+    return children;
+  },
+}));
+
+const FIXTURE_WORKFLOW = {
+  id: 1,
+  stages: [
+    {
+      id: 1,
+      name: 'stage-1',
+    },
+  ],
+};
+
+const server = setupServer(
+  rest.get('*/review-workflows/workflows', (req, res, ctx) => {
+    return res(
+      ctx.json({
+        data: [FIXTURE_WORKFLOW],
+      })
+    );
+  })
+);
```

Comment:
This is a question for bigger discussion, but why do we declare a server _everywhere_ why not just use one?

Reply:
I think it makes sense to have them as part of each test, because every setup is very individual (especially when it comes to error handling). Now, we could talk about having a global admin API server, that is extensible ... I'd like that, but I am worried it won't be useful because different tests may need different schemas.

Reply:
yeah sounds good, I think MSW will allow us to do something smart, just need to brush up on the docs! 😄